packages/ui/src/components/vuestic-components/va-select/VaSelect.vue

```diff
@@ -93,27 +92,30 @@
       </template>
 
       <!-- Stop propagation for enter keyup event, to prevent VaDropdown closing -->
-      <va-dropdown-content @keyup.enter.stop>
+      <va-dropdown-content
+        @keyup.enter.stop
+        @keydown.esc.prevent="hideAndFocus"
+        @keydown.tab="hideDropdown"
+      >
+        <va-input
+          v-if="showSearchInput"
+          :id="$props.id"
+          ref="searchBar"
+          v-model="searchInputValue"
+          class="va-select__input"
+          placeholder="Search"
+          removable
+          :name="$props.name"
+          :tabindex="tabindex + 1"
+          :bordered="true"
+          @keydown.up.stop.prevent="hoverPreviousOption"
```

Comment:
`="hoverPreviousOption()"` adding brackets is preferable.
Makes it explicit you don't pass anything to method.